slda.cpp

```diff
@@ -986,6 +989,7 @@ void slda::infer_only(corpus * c, const settings * setting, const char * directo
             {
                 score += eta[i][k] * phi_m[k];
             }
+	    fprintf(inf_score_file, "%5.5f ", score);
```

Comment:
suggest also adding doc id for the easy output understanding.
indent is not correct.